src/Queue.php

```diff
@@ -94,29 +205,96 @@ public function run(Job $job) {
                 $id = $job->route;
             }
             \Yii::error("Fatal Error: Error running route '{$id}'. Message: {$e->getMessage()}", 'yii2queue');
-            throw new \yii\base\Exception("Error running route '{$id}'. Message: {$e->getMessage()}. File: {$e->getFile()}[{$e->getLine()}]. Stack Trace: {$e->getTraceAsString()}", 500);
+            throw new \yii\base\Exception(
```

Comment:
Agree, since there is a new method to `release` to handle queue like DB and Redis, we should have an error handler for this. So there is three types of job execution: 1) success (`true`), 2) fail (`false`) and 3) error (`Exception`).